pom.xml

```diff
@@ -0,0 +1,57 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.sps</groupId>
+  <artifactId>portfolio</artifactId>
```

Comment:
Should probably change this to something more relevant

Reply:
yup oki, any suggestions? aq map?

Reply:
In future please ensure all review comments are resolved before adding Hosts for final approval.

Reply:
Noted, Sorry Sai!

Reply:
@purple-waves  changed name to AQ_Map